src/containers/Login/Login.js

```diff
@@ -0,0 +1,165 @@
+import React, { Component } from "react";
+import { connect } from "react-redux";
+import { push } from "connected-react-router";
+import TextField from "@material-ui/core/TextField";
+import Button from "@material-ui/core/Button";
+import styled from "styled-components";
+import { routes } from "../Router";
+import { Card } from "@material-ui/core";
+// import { login } from "../../actions/auth";
+import InputAdornment from '@material-ui/core/InputAdornment';
```

Comment:
Caso não for utilizar mais esse código comentado, seria legal tirar ele.

Reply:
Feito!